docs/5_2_scheduled_execution_api.adoc

```diff
@@ -103,33 +106,31 @@ The <<fmi3ActivateModelPartition>> function has the following arguments:
 
 * [[clockReference,`clockReference`]] `clockReference`: <<valueReference>> of the <<inputClock>> associated to the model partition which shall be activated.
 
-* [[clockElementIndex,`clockElementIndex`]] `clockElementIndex`: The element index of the <<Clock>> variable associated to the model partition which shall be activated.
-For a scalar <<Clock>> variable this must be 0; for array <<Clock>> variables, the element clock is specified using the 1-based element index.
+* [[clockElementIndex,`clockElementIndex`]] `clockElementIndex`: The element index of the <<Clock>> variable associated with the model partition which shall be activated.
+For a scalar <<Clock>> variable the index must be 0; for an array <<Clock>> variable, the index is 1-based.
 Using the element index 0 means all elements of the <<Clock>> variable.
 (Note: If an array has more than one dimension the indices are serialized in the same order as defined for values in <<serialization-of_variables>>).
 
 * [[activationTime, `activationTime`]] `activationTime`: value of the <<independent>> variable of the assigned <<Clock>> tick time latexmath:[\mathbf{t}_i] _[typically: simulation (i.e. virtual) time]_ (which is known to the simulation algorithm).
```

Comment:
_[typically: simulation (i.e. virtual) time]_ 

I thought real-time applications are the most common ones, and virtual time is the exception